jobs/build-arch.Jenkinsfile

```diff
@@ -326,6 +326,17 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             }
         }
 
+        // IBM Crypto Express Card test. 
+        // Starting with 4.17.
+        def cex_rhcos_version = ""
+        cex_rhcos_version = shwrapCapture("cosa meta --get-value buildid | cut -c-3")
+        if (cex_rhcos_version >= 417){
+            stage("Kola:Cextest") {
+                kola(cosaDir: env.WORKSPACE, arch: basearch, skipUpgrade: true,
+                     extraArgs: "--tag cextest")
+            }
+        }
+
```

Comment:
Should be able to drop this once we do https://github.com/coreos/coreos-assembler/pull/3828#discussion_r1666017380

Reply:
will do